packages/extension-polkagate/src/popup/crowdloans/contribute/Review.tsx

```
@@ -1,30 +1,29 @@
 /* eslint-disable react/jsx-max-props-per-line */
 /**
  * @description
  * this component opens contribute to crowdloan review page
  * */
 
+import type { LinkOption } from '@polkagate/apps-config/endpoints/types';
+import type { ApiPromise } from '@polkadot/api';
 import type { Balance } from '@polkadot/types/interfaces';
+import type { AccountId } from '@polkadot/types/interfaces/runtime';
+import type { BN } from '@polkadot/util';
+import type { Crowdloan, Proxy, ProxyItem, TxInfo } from '../../../util/types';
```

Biome

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)